src/clients/c++/request.h

```diff
@@ -401,7 +380,11 @@ class InferContext {
       /// CLASS format is the top-k highest probability values of the
       /// result and the associated class label (if provided by the
       /// model).
-      CLASS = 1
+      CLASS = 1,
+
+      /// SHARED_MEMORY format, the entire result tensor of values are written
```

Comment:
I don't think we need this. The format of the result is still "RAW" even if it is in shared memory.